src/RapidezServiceProvider.php

```diff
@@ -154,33 +153,11 @@ protected function bootRoutes(): self
         return $this;
     }
 
-    protected function registerThemes(): self
-    {
-        if (app()->runningInConsole()) {
-            return $this;
-        }
-
-        $path = config('rapidez.frontend.themes.' . request()->server('MAGE_RUN_CODE', request()->has('_store') && ! app()->isProduction() ? request()->get('_store') : 'default'), false);
-
-        if (! $path) {
-            return $this;
-        }
-
-        config([
-            'view.paths' => [
-                $path,
-                ...config('view.paths'),
-            ],
-        ]);
-
-        return $this;
-    }
-
     protected function bootViews(): self
     {
         $this->loadViewsFrom(__DIR__ . '/../resources/views', 'rapidez');
 
-        View::composer('rapidez::layouts.app', ConfigComposer::class);
+        View::composer('rapidez::layouts.config', ConfigComposer::class);
```

Comment:
Does it make sense to still use a composer for this? Maybe a controller is a better idea?